code/modules/integrated_electronics/subtypes/arithmetic.dm

```diff
@@ -311,3 +311,34 @@
 	set_pin_data(IC_OUTPUT, 1, result)
 	push_data()
 	activate_pin(2)
+
+// -Max- //
+/obj/item/integrated_circuit/arithmetic/max
+	name = "max circuit"
+	desc = "This circuit sends out the highest number."
+	extended_desc = "The highest number is put out. Null is ignored."
+	icon_state = "addition"
+	var/state=0
+	spawn_flags = IC_SPAWN_DEFAULT|IC_SPAWN_RESEARCH
```

Comment:
New vars should be at the bottom of datum definitions

Also `state` doesn't accurately describe what the var does. Code is free, name it something more verbose like `min_comparison`.

Use `FALSE` instead of `0` as well